tests/providers/microsoft/azure/hooks/test_wasb.py

```diff
@@ -154,6 +166,11 @@ def test_managed_identity(self):
         assert isinstance(hook.get_conn(), BlobServiceClient)
         assert isinstance(hook.get_conn().credential, DefaultAzureCredential)
 
+    def test_ad_connection(self):
```

Comment:
can you please improve the naming for this `test_ad_connection`

Reply:
Thanks for pointing out. Changed it to `test_azure_directory_connection`.